packages/material-ui-styles/test/index.spec.tsx

```diff
@@ -76,6 +76,7 @@ import styled, { StyledProps } from '@material-ui/styles/styled';
   const useUnsafeProps = makeStyles(
     createStyles({
       root: (props: any) => ({
+        // tslint:disable-next-line no-unsafe-any
```

Comment:
Is it not the line above that's using an unsafe any?

Reply:
It's usage i.e. access. But I'm changing it to only be disabled in the specific test.

Reply:
Should this change be reverted? The other changes seem to be scoped to the icons package.